kubernetes-controllers/node-exporter-daemonset.yaml

```
@@ -0,0 +1,25 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: node-exporter
+  labels:
+    app: node-exporter
+spec:
+  selector:
+    matchLabels:
+      app: node-exporter
+  template:
+    metadata:
+      labels:
+        app: node-exporter
+    spec:
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          effect: NoSchedule
```

Вы начали копать в нужную сторону, но не совсем верно определили tolerations.
Предлагаю вам подробнее ознакомиться с документацией:
https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/#concepts -> секция Note и с официальным манифестом для node exporter.

---

```suggestion
        - operator: "Exists"
```

@Jasstkn, спасибо, кажется догадался. В таком варианте получается, что exporter развернется на всех нодах кластера? вне зависимости мастер/воркер ([#mark-the-node-as-control-plane](https://kubernetes.io/docs/reference/setup-tools/kubeadm/implementation-details/#mark-the-node-as-control-plane))

---

@dmitriye да, все верно. Таким образом вы будете запускать нод экспортер на всех нодах в кластере - может быть полезно, чтобы каждый раз не указывать специфичные толерейшнс.